charts/resources/instance/templates/instance.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: servicecatalog.k8s.io/v1alpha1
+kind: Instance
+metadata:
+  name: {{ .Values.name }}
+spec:
+  serviceClassName: {{ .Values.serviceClassName }}
+  planName: {{ .Values.planName }}
```

Comment:
@vaikas-google 
>Could we add parameters here to see how they would line up?

are you asking for a list of params in the README for this chart?

>Right now this seems like a lot of overhead to create a chart and a helm install, when a simple kubectl create would suffice.

It is overkill right now for sure. This PR is preparing for the following situation, which is coming in the near future:

- We are behind the aggregator
- A new user wants a quick way to add a broker, provision, or bind

This chart will provide an easy way to quickly do operations against our API.

Let me know what you think @vaikas-google